:construction:  Toggle SCIPY_OPENBLAS dependency when installing from src (#1220)
PennyLaneAI/pennylane-lightning: .github/CHANGELOG.md

<h3>Internal changes ⚙️</h3>

> - Toggle the `scipy-openblas32` dependency when installing from source using the `ENABLE_SCIPY_OPENBLAS` compilation flag.

Reviewer: ```suggestion
- `scipy-openblas32` dependency can be set manually when installing from source using the `ENABLE_SCIPY_OPENBLAS` cmake flag.
```
